Address the feedback in https://github.com/Tao-VanJS/twikoo-cloudflare/issues/3

Tao-VanJS opened the pull request.

imaegoo commented:
test passed

![image](https://github.com/Tao-VanJS/twikoo-cloudflare/assets/20182252/823f7ea5-7429-4be5-9c82-46ae0337c5fb)

Tao-VanJS commented:
奇怪，我在运行`npx wrangler deploy --minify`还是会遇到这样的错误：

```
✘ [ERROR] Could not resolve "tencentcloud-sdk-nodejs"

    node_modules/twikoo-func/utils/lib.js:55:33:
      55 │ ...entcloud = require('tencentcloud-sdk-nodejs') // 腾讯云 API ...
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "tencentcloud-sdk-nodejs" as external to exclude it from the bundle, which will remove this error. You can also surround this "require" call with a try/catch block to handle this failure at run-time instead of bundle-time.


✘ [ERROR] Build failed with 1 error:

  node_modules/twikoo-func/utils/lib.js:55:33: ERROR: Could not resolve "tencentcloud-sdk-nodejs"
```

imaegoo commented:
You can mark the path "tencentcloud-sdk-nodejs" as external to exclude it from the bundle,
我觉得应该试试这句话

Tao-VanJS commented:
> You can mark the path "tencentcloud-sdk-nodejs" as external to exclude it from the bundle, 我觉得应该试试这句话

这句话让我很困惑。我没有找到怎么可以把一个path mark成"external"的方法。所以我只知道用`try {...} catch {...}`包起来的办法。另外我看到您已经可以把这个repo成功部署到Cloudflare worker上去了。能分享一下怎么从一个**clean**的状态部署上去的步骤吗？目前我把这个PR合并进去再进行部署就会出错:-(现在无法完成一个成功的部署。

imaegoo commented:
> 能分享一下怎么从一个clean的状态部署上去的步骤吗？

和README中的步骤基本一样，不同之处是我把第二步执行的命令换成了

```shell
echo "" > node_modules/tencentcloud-sdk-nodejs/tencentcloud/index.js
```

Tao-VanJS commented:
> > 能分享一下怎么从一个clean的状态部署上去的步骤吗？
> 
> 和README中的步骤基本一样，不同之处是我把第二步执行的命令换成了
> 
> ```shell
> echo "" > node_modules/tencentcloud-sdk-nodejs/tencentcloud/index.js
> ```

啊，这个很聪明！我把同样的技巧也用在了jsdom上面。这个降低尺寸的效果比较之前的`rm -rf ...`还要好很多，因为顺便也把所有依赖的包也一并移除了。目前我部署的脚本只有不到400kB，和1MiB的上限还有很大的距离！

ChaiByte commented:
It seems unnecessary to fork a new branch to exist independently, which would affect version synchronization. Consider migrating the original project to the organization, or the author could serve as the maintainer of the official forked version for subsequent development.

Tao-VanJS commented:
Hi @ChaiByte,

I don't fully understand what you meant. Are there any specific actions you suggest me to do?

ChaiByte commented:
> Hi @ChaiByte,
> 
> I don't fully understand what you meant. Are there any specific actions you suggest me to do?

I have envisioned a workflow, which is to directly set you as a **collaborator** of the twikoo-cloudflare repository. This way, you can directly obtain the permission to push new branches to twikoo-cloudflare without having to consider the complex upstream and downstream relationships between the two repositories. In fact, it might be that I am overthinking it.

- From: Tao-VanJS:main -> twikoojs:main
- To: twikoojs:feat-xxx -> twikoojs:main

I mistakenly introduced a strong assumption that you might be the long-term maintainer.

The reason for my confusion is that I am currently using R2 and came across your project, which is very useful. However, I noticed that your main branch is more up-to-date than the official fork, so for a moment, I was unclear about the collaboration method you are using. Perhaps I just need to focus on the official fork version, right?

Tao-VanJS commented:
Hi @ChaiByte,

I'm not the long-term maintainer of the twikoo project. I'm a user of twikoo, but I found that the existing cloud deployments of twikoo had the issues of long cold-start time (it takes about `6s` to initially load the comments if the site hasn't been visited for a while). So I built the Cloudflare-based solution, which reduced the cold-start time to `0.5s` thanks to the seamless integration between Cloudflare workers and D1. After building my solution I told the maintainer of twikoo - @imaegoo: feel free to take my code to the twikoo project. I don't mind if the maintainer puts my code into the main repo: https://github.com/twikoojs/twikoo, or creates a separate repo for it. It's the maintainer's decision. This is why you're seeing the situation as it is.

Some GitHub user went to my clone of the repo and provided some feedback, I addressed the feedback in my repo and created this PR to patch into the official repo of twikoo-cloudflare.
